pkg/agents/team3/foodCalculations.go

```diff
@@ -93,12 +147,11 @@ func (a *CustomAgent3) takeFoodCalculation() int {
 		return foodToEat
 
 	default: //uses foodToEat and foodToLeave
-
 		if platFood-foodToLeave >= foodToEat {
 			return foodToEat
 		} else if platFood >= foodToLeave { //e.g. platFood=15, foodToLeave=10, foodToEat=10 --> if morality=100, take 5, if morality=0, take 10
```

Comment:
Technically doesn'tneed to be else if, can just be if

Reply:
Is an else if less efficient than an if? Can change as well but personally don't think its necessary

Reply:
It's more cos you return earlier, so it's fine to just be an if